docs/ossm/ossm2-migration/torubleshooting/README.md

```diff
@@ -0,0 +1,49 @@
+
+To workaround this problem it's necessary to restart the 2.6 control plane so 3.0 control plane becomes a leader and creates the `istio-ca-root-cert` ConfigMap.
+
+## Migrated workload fails to start
+The init container is failing with:
+
+    `Failed to create pod sandbox: rpc error: code = Unknown desc = failed to create pod network sandbox k8s_httpbin-5746ccddc6-tgvqz_httpbin-3-strict_db236765-6ccd-4dbe-a079-19b5f878f329_0(5c5ffe21391046b205b002cb35ee2ccb17badf5bfab3942a0cae1db1a1d5b3f4): error adding pod httpbin-3-strict_httpbin-5746ccddc6-tgvqz to CNI network "multus-cni-network": plugin type="multus-shim" name="multus-cni-network" failed (add): CmdAdd (shim): CNI request failed with status 400: 'ContainerID:"5c5ffe21391046b205b002cb35ee2ccb17badf5bfab3942a0cae1db1a1d5b3f4" Netns:"/var/run/netns/0938ea9a-401d-484b-b174-53f987811d6f" IfName:"eth0" Args:"IgnoreUnknown=1;K8S_POD_NAMESPACE=httpbin-3-strict;K8S_POD_NAME=httpbin-5746ccddc6-tgvqz;K8S_POD_INFRA_CONTAINER_ID=5c5ffe21391046b205b002cb35ee2ccb17badf5bfab3942a0cae1db1a1d5b3f4;K8S_POD_UID=db236765-6ccd-4dbe-a079-19b5f878f329" Path:"" ERRORED: error configuring pod [httpbin-3-strict/httpbin-5746ccddc6-tgvqz] networking: [httpbin-3-strict/httpbin-5746ccddc6-tgvqz/db236765-6ccd-4dbe-a079-19b5f878f329:v2-6-istio-cni]: error adding container to network "v2-6-istio-cni": exit status 1 ': StdinData: {"auxiliaryCNIChainName":"vendor-cni-chain","binDir":"/var/lib/cni/bin","clusterNetwork":"/host/run/multus/cni/net.d/10-ovn-kubernetes.conf","cniVersion":"0.3.1","daemonSocketDir":"/run/multus/`
+
+This error is visible when both control planes are trying to inject the side car proxy. To avoid this, `maistra.io/ignore-namespace: "true"` must be added. This label disables sidecar injection for OpenShift Service Mesh 2.6 proxies in the namespace. With the label applied, OpenShift Service Mesh 2.6 stops injecting proxies in this namespace, and any new proxies are injected by OpenShift Service Mesh 3.0. Without this label, the OpenShift Service Mesh 2.6 injection webhook tries to inject the pod and the injected sidecar proxy refuses to start since it will have both the OpenShift Service Mesh 2.6 and the OpenShift Service Mesh 3.0 Container Network Interface(CNI) annotations.
+
+## Migrated workload is not injected at all
+Migrated workload is running without a side car:
+```sh
+oc get pods -n bookinfo
+NAME                                      READY   STATUS    RESTARTS   AGE
+productpage-v1-7559db9df5-xgp7j           1/1     Running   0          23m
+```
+
+1. Verify that injection labels are set correctly on the namespace and/or deployments:
+
+    `istio-injection=enabled` label works only when the IstioRevision is called `default` or the `default` IstioRevisionTag exists. In other cases, `istio.io/rev` label must be used.
+1. When using `istio.io/rev` label, make sure the namespace is not labeled also with `istio-injection=enabled` as `istio-injection` takes precedence over `istio.io/rev` label
```

Comment:
```suggestion
1. Avoid conflicting labels: 

If the namespace has both `istio-injection=enabled` and `istio.io/rev` labels, the `istio-injection` label will take precedence. To prevent conflicts, remove the `istio-injection` label when using `istio.io/rev`
```